README.md

```diff
@@ -22,6 +22,7 @@ docker compose -f docker-compose-dev.yml build
 docker compose -f docker-compose-dev.yml up -d
 docker compose -f docker-compose-dev.yml run web bin/rails db:create # ignore any errors here
 docker compose -f docker-compose-dev.yml run web bin/rails db:migrate db:seed
+docker compose -f docker-compose-dev.yml run web bin/rails assets:precompile
```

Comment:
without precompiling the assets, tailwind will not work so it's essential to have this step.

Reply:
Were you not seeing assets when running this locally? This should already be being executed during the build [from the Dockerfile.dev](https://github.com/goinvo/staffplan_redux/blob/4648d622a7dd29629269fd2fc4d91dfaaaa93951/Dockerfile.dev#L36).

Reply:
Scratch what I said, this is 💯 needed. I must have had a previous build on my filesystem while I put this together 🙇🏻